crates/aptos/src/node/mod.rs

```diff
@@ -381,20 +337,14 @@ impl CliCommand<Transaction> for RegisterValidatorCandidate {
                 }
             };
 
+        // Double BCS encode, so that we can hide the original type
```

Comment:
Can you put this back down next to the actual double encoding?

Reply:
oh good call